src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -95,7 +93,7 @@ const CustomDescription = ({
   data, value = 0, recipient, classes,
 }: CustomDescProps) => (
   <>
-    <Paragraph noMargin data-testid={TRANSACTIONS_DESC_CUSTOM_VALUE_TEST_ID}>
+    <Block data-testid={TRANSACTIONS_DESC_CUSTOM_VALUE_TEST_ID}>
```

Comment:
same here

Reply:
It has the component `EtherscanLink`, which has the component `Block` as his root element